utils/get-top-position.js

```diff
@@ -0,0 +1,35 @@
+/*
+ * offset - distance from clicked element and container with relative position
+ * top - distance from clicked element to top of window
+ * itemHeight - height of the item that will be positioned
+ * containerHeight - height of the container with relative position
+ * adjustment - adjust offset of clicked element (used for some cases)
+*/
+
+export default function(offset, top, itemHeight, containerHeight, adjustment) {
```

Comment:
what if we used a single `options` argument so if we ever want to set things like padding through the function, it's easy to update.